### PR TITLE
Add Vue.Draggable@2.11.0 w/ git auto-update

### LINKED ISSUE #10659: [Request] Add Vue.Draggable
**Library name: Vue.Draggable** 
**Git repository url: https://github.com/SortableJS/Vue.Draggable**
**npm package url(optional):** 
**License(s): MIT**
**Official homepage: https://github.com/SortableJS/Vue.Draggable**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dakshshah96**
close #10659, cc @David-Desmaisons

Pull request for issue: #10659

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/SortableJS/Vue.Draggable
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/vuedraggable
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 40
   - Count of stars: 978
   - Count of forks: 95
 * NPM download stats (optional):
   - Downloads in the last day: 637
   - Downloads in the last week: 3,152
   - Downloads in the last month: 11,853

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/vuedraggable/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "vuedraggable",
+  "version": "2.11.0",
+  "filename": "vuedraggable.min.js",
+  "description": "Vue component allowing drag-and-drop sorting in sync with View-Model. Based on Sortable.js.",
+  "keywords": [
+    "vue",
+    "vuejs",
+    "drag and drop",
```
Comment (dakshshah96) [inline ajax/libs/vuedraggable/package.json]: I don't think the keyword `and` makes sense simply on its own. Let's not use that?

**Comment by dakshshah96** [comment]
@extend1994 Please add `in progress` label here.

**Comment by PeterDaveHello** [comment]
@dakshshah96 better tag @qwe19272375 to do it

File: ajax/libs/vuedraggable/package.json (hunk 1 above)
Comment (extend1994) [inline ajax/libs/vuedraggable/package.json]: I am not pretty sure about this. @cdnjs/intern3 Can anyone help me with this?

**Comment by sufuf3** [comment]
ping @extend1994 @hare1039 @clairetsai818 

Note: I think the old versions can add later after this PR close.

**Comment by PeterDaveHello** [comment]
@sufuf3 please open a new issue to request its old versions.

**Pull request merged**